CONTRIBUTING-detailed-instruction.md

```diff
@@ -0,0 +1,100 @@
+	 * [new branch]      master     -> upstream/master
+	# merge upstream/master into local master
+	$ git checkout master
+	$ git merge upstream/master
+	# push changes to your forked repository
+	$ git push origin
+	```
+2. **Create a topic branch** on your local repository.
+	1. From the repository directory
+	2. `git checkout master` to make master the base
+	3. `git checkout -b TOPIC-NAME`, where TOPIC-NAME is an appropriate name for branch, e.g "contributing" 
+3. **Make your changes**: hack, hack, edit, hack, edit, ...
+4. **Add and commit** changes to your topic branch. Refer to the [Git Documentation](https://git-scm.com/docs) for details on [`add`](https://git-scm.com/docs/git-add) and [`commit`](https://git-scm.com/docs/git-add) usage:
+	1. Use the `git add` command to add content to the index that defines the snapshot that will be committed,
+	2. Use the `git commit` command to commit the index to the current working branch, or
+	3. Use `git commit -a` to add and commit all changes.
+	```bash
+	$ git add *                # adds all changes to index
+    $ git add CONTRIBUTING.md  # add single file to index
+	$ git commit -a            # adds all changes and commits
+	$ git commit -m "<commit message line here>"
+	```
+5. **Push your changes** to your forked repository. The first time you push your branch use the `git push -u` command as shown below. Subsequent pushes after further edits can use `git push origin` (assumes you are working in branch on local repository).
+	
+	```bash
+	# initial push of topic branch
+	$ git push -u origin your-topic-branch  # origin is your forked repository
+	# pushes of topic branch after further edits and commits
+	$ git push origin                       # origin is your forked repository
+	```
+6. **Review your change** as it appears on your fork of the scalabridge/curriculum repository. **Make any additional corrections** as needed by repeating steps 3-5 above.
+7. [**Make your pull request**](https://help.github.com/articles/creating-a-pull-request-from-a-fork/)
+	1. Go to your forked repository on GitHub
+  	2. To the right of the Branch menu, click **New pull request**.
+  	3. On the Compare page, click **compare across forks**.
+  	4. Confirm that the *base fork* is the repository you'd like to merge changes into. Use the *base branch* drop-down menu to select the branch of the upstream repository you'd like to merge changes into.
+  	5. Use the *head fork* drop-down menu to select your fork, then use the *compare branch* drop-down menu to select the branch you made your changes in.
+  	6. Type a title and description for your pull request. 
+  	7. If you would like to allow anyone with push access to the upstream repository to make changes to your PR, then select Allow edits from maintainers.
+  	8. Click **Create pull request**.
+8. Pull request review by ScalaBridge team members.
+	1. Make revisions as necessary
+9. Squash commits and merge into master branch of scalabridge repository. 
```

Comment:
"master branch" should be "destination branch", since the instructions don't specify that the base branch will always be master.